packages/crayons-core/src/components/icon/readme.md

```diff
@@ -1,13 +1,23 @@
 # Icon (fw-icon)
-fw-icon displays an icon-sized image that imparts meaning to the component it is associated with.
+fw-icon is a renderer of an svg file which displays an icon-sized image that imparts meaning to the component it is associated with.
+We have covered following topics in the implementation of fw-icon as a Icon Library.
 
-## Demo
+1. Enable fw-icon as an optimized renderer for a SVG with inbuilt capabilities like Intersection-Observer and Fetch-API Memoization.
+   Go through the docs to undertand the various props it supports.
+2. React-Support as part of Caryons Release. 
```

Comment:
Please fix all the typos in this readme

Reply:
closed

Reply:
I still see the typo here for Crayons. Can you please check it ?